docs/apim/4.0/guides/gravitee-expression-language.md

```diff
@@ -193,6 +193,17 @@ Request context attributes and examples are listed below.
 {% endtab %}
 {% endtabs %}
 
+#### Important Note for v4 Emulation Engine Users
+When using the v4 emulation engine, it is important to note that the api-key attribute is no longer accessible in the context if you wish to use it as a logging filter condition. Instead of referencing the api-key attribute directly with expressions like `{#context.attributes['api-key'] == 'api-key-value'}`, users can retrieve the API key from request parameters or headers.
+
+To achieve this, consider using the following expression:
```

Comment:
Instead of "To achieve this, consider using the following expression", I would say: "To retrieve API key from the request parameters or headers, use the following expression:

Reply:
Done